### PR TITLE
#1733 payout e2e tests

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: app/test-e2e/eto/InvestWithRegularEther.spec.e2e.ts [hunk 1]
```diff
@@ -46,10 +47,10 @@ describe("Invest with ethereum", () => {
       cy.get(tid("portfolio-reserved-asset-token-price")).then($element => {
         const neuReward = parseAmount($element.text());
 
-        cy.get<number>("@estimatedReward").then(estimatedReward => {
+        cy.get<BigNumber>("@estimatedReward").then(estimatedReward => {
```
Comment (Moejoe90) [inline app/test-e2e/eto/InvestWithRegularEther.spec.e2e.ts]: :+1:

File: app/test-e2e/utils/assertions.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
+import { get } from "lodash";
```
Comment (Moejoe90) [inline app/test-e2e/utils/assertions.ts]: :+1:

File: app/test-e2e/utils/index.ts [hunk 3]
```diff
@@ -49,62 +31,18 @@ export const goToProfile = () => {
   cy.visit("/profile");
 };
 
-export const clearEmailServer = () => {
-  cy.request({ url: mockApiUrl + "sendgrid/session/mails", method: "DELETE" });
+export const goToPortfolio = () => {
```
Comment (Moejoe90) [inline app/test-e2e/utils/index.ts]: Maybe in the future, this can go to another folder `routings` or something?

Comment (desfero) [reply app/test-e2e/utils/index.ts]: yes, would be really good for index.ts to only contain reexports

**Comment by Desperado** [comment]
Looks very good :100:

**Pull request merged**